api/src/wfl/service/datarepo.clj

```diff
@@ -97,6 +97,15 @@
   ([job-id]
    (poll-job job-id 5)))
 
+(defn job-done?
+  "Check the job with `job-id` to see if it's done yet."
+  [job-id]
+  (let [result   #(get-repository-json "jobs" job-id "result")
+        running? #(-> (get-repository-json "jobs" job-id)
+                      :job_status
+                      #{"running"})]
+    (if (running?) false (result))))
```

Comment:
If you tweaked `running?` to test if it were done, the following might be more natural
`(and (done?) (result))`